cadc-storage-adapter-swift/src/intTest/java/org/opencadc/inventory/storage/swift/SwiftByteRangeTest.java

```diff
@@ -109,4 +109,9 @@ public void cleanupBefore() throws Exception {
         }
         log.info("cleanupBefore: DONE");        
     }
+    
+    @Test
+    public void testCleanupOnly() {
+        log.info("testCleanupOnly: no-op");
```

Comment:
What is this for?

Reply:
A no-op task allows one to run just this test method so you get the ctor (init) and cleanup code and no other tests. It's handy if you are having trouble with init (connection, auth, the init itself, or the cleanupBefore code).